lighthouse-treemap/app/src/main.js

```diff
@@ -458,10 +478,11 @@ class TreemapViewer {
     ];
 
     if (bytes !== undefined && total !== undefined) {
-      let str = `${TreemapUtil.formatBytes(bytes)} (${Math.round(bytes / total * 100)}%)`;
+      const percentStr = `${Util.i18n.formatNumber(Math.round(bytes / total * 100))}%`;
```

Comment:
is concating % here OK?

Reply:
for plain % (not concatenating to another string), there's `new Intl.NumberFormat(locale, {style: 'percent'}).format(value)`. You have to be careful with report formatting stuff because Safari formatting support has really lagged the other browsers (and Node), but `'percent'` has been supported for a while now.

Reply:
but then this value isn't just ##%, it's put into ``let str = `${TreemapUtil.formatBytes(bytes)} (${percentStr})`;``, so agree with the 🤷 :)